<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оформление заказа</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0" colspan="4">
        <widget class="QTableWidget" name="carpet_table"/>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="add_b">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="del_b">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="back_b_2">
         <property name="text">
          <string>В меню</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="order_b">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Добавить в заказ</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QListWidget" name="carpet_list">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>90</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>621</width>
           <height>81</height>
          </size>
         </property>
         <property name="title">
          <string>Ковер</string>
         </property>
         <widget class="QComboBox" name="carpet_cb">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>20</y>
            <width>201</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Вид изделия</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>50</y>
            <width>51</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Ширина</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="w_sb">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>50</y>
            <width>51</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>50</y>
            <width>47</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>м</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="h_sb">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>50</y>
            <width>51</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>50</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Высота</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>50</y>
            <width>47</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>м</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>50</y>
            <width>31</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>S =</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>50</y>
            <width>31</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>м²</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="s_l">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>50</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>19</y>
            <width>51</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Цена 1 м²</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>50</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Цена стирки</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="pricem_l">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>17</y>
            <width>71</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>20</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>руб.</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="resprice_l">
          <property name="geometry">
           <rect>
            <x>510</x>
            <y>50</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>70</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="4">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>641</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string>Клиент</string>
         </property>
         <widget class="QLineEdit" name="name_l">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>20</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_21">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>47</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>ФИО:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_22">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>47</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Телефон:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="phone_l">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>50</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
          <property name="inputMask">
           <string>+00000000000</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_23">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>12</y>
            <width>47</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Адрес:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="address_l">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>20</y>
            <width>341</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QTableWidget" name="carpet_table_2"/>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Дата Выдачи:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QDateEdit" name="dateEdit">
         <property name="date">
          <date>
           <year>2023</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="price_l">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Общая стоимость:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="back_b">
         <property name="text">
          <string>Назад</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QPushButton" name="create_ord_b">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Создать заказ</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
